packages/devtools_app/lib/src/screens/inspector_v2/layout_explorer/ui/utils.dart

```diff
@@ -441,3 +563,47 @@ class LayoutExplorerBackground extends StatelessWidget {
     );
   }
 }
+
+/// Builds and positions a label for the [LayoutExplorerBackground] as
+/// determined by the widget's padding.
+class PositionedBackgroundLabel extends StatelessWidget {
+  const PositionedBackgroundLabel({
+    super.key,
+    required this.labelText,
+    required this.labelColor,
+    required this.topPadding,
+    required this.bottomPadding,
+    required this.leftPadding,
+    required this.rightPadding,
+  });
+
+  final String labelText;
+  final Color labelColor;
+  final bool topPadding;
+  final bool bottomPadding;
+  final bool leftPadding;
+  final bool rightPadding;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        // Push to the bottom if there is no padding on the top.
+        if (!topPadding && bottomPadding) const Spacer(),
+        Row(
+          children: [
+            // Push to the right if there is no padding on the left.
+            if (!leftPadding && rightPadding) const Spacer(),
```

Comment:
Ditto

Reply:
Same thing, if we have both then the label can stay on the left.